### PR TITLE
logout page

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
added logout page.

Site Link: https://zen-golick-1606c1.netlify.app/store-pages/logout-page/logout.html

### CODE REVIEW

**Review by malaykhakhar — APPROVED** [review]
The code looks good. No suggestions. You can merge the code.

File: store-pages/logout-page/logout.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@import "../../common.css";
+
+.logout-flex {
```
Comment (git-0r) [inline store-pages/logout-page/logout.css:3]: suggestion:- here you can extract the common properties of "logout-flex" & create different classes for flex-direction.

File: store-pages/logout-page/logout.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    <i class="fas fa-shopping-cart icon-badge"></i>
+                    <span class="badge badge-online badge-to-icon">9</span>
+                </div>
+            </a>
+        </div>
+    </header>
+    <!-- Header Section Ends -->
+
+    <!-- Main Section Starts -->
+    <div class="logout-flex">
```
Comment (git-0r) [inline store-pages/logout-page/logout.html:43]: if this is your main section you can use "main"  element.

**Review by git-0r — APPROVED** [review]

File: store-pages/logout-page/logout.css (hunk 1 above)
Comment (yaswanthmyneni) [reply store-pages/logout-page/logout.css:3]: I will do it later.

**Pull request merged**